### ISSUE TITLE
Add support to not clean column names (preserve same column names)

### ISSUE THREAD

**Issue by christian-heins-velero** [jdglaser/fast-to-sql#21]
I am triying to upload a dataframe to a microsoft sql server. 

The issue is the column names will be cleaned and adjujsted every time the table is uploaded. Adding an option to leave the same column names would be amazing. 

I know that when we send the column names to the sql server, they can be intepreted weirdly. I recently found out that by adding `` (Backwardstrings) in the insert statement to the column names when they contain special character they can be read correctly by the sql server.

**Comment by jdglaser** [comment]
This was addressed with the most recent release [v2.3.0](https://github.com/jdglaser/fast-to-sql/releases/tag/v2.3.0). You should now be able to pass the parameter `clean_cols=False` to the function and columns will not be cleaned. **Note**: if you do this, it is up to you to make sure that the column names in your DataFrame are compatible with SQL server.

**Comment by christian-heins-velero** [comment]
Hi Jared!

I updated just now your library and the option to not clean the tables is not working as of now, i had to implement the following function called update_column_names() in order to keep my column names original:

Traceback (most recent call last):
  File "test.py", line 1008, in <module>
    main()
  File "test.py", line 927, in main
    fast_sql_test()
  File "test.py", line 861, in fast_sql_test
    create_statement = fts(
                       ^^^^
  File "C:\AppData\Roaming\Python\Python311\site-packages\fast_to_sql\fast_to_sql.py", line 80, in fast_to_sql
    cur.execute(create_statement)
pyodbc.ProgrammingError: ('42000', '[42000] [Microsoft][ODBC Driver 18 for SQL Server][SQL Server]Falsche Syntax in der Nähe von "2". (102) (SQLExecDirectW)')


<img width="932" alt="Screenshot 2024-03-27 153247" src="https://github.com/jdglaser/fast-to-sql/assets/149429263/c593695a-47ed-400b-aa93-63272e8a94fc">
<img width="743" alt="Screenshot 2024-03-27 153600" src="https://github.com/jdglaser/fast-to-sql/assets/149429263/41883de6-5905-42ef-a476-909761850978">

```
      def fast_sql_test():
        from module_connections import connection_pyodbc_module
        from fast_to_sql import fast_to_sql as fts

        table_name =  "my_great_table"
        database = "Test_Christian"
        connection = connection_pyodbc_module(database=database)
        cursor =  connection.cursor()

        df = pd.DataFrame({
            "Col1": [1, 2, 3],
            "Col 2": ["A", "B", "C"],
            "Col3": [True, False, True],
            "Col 4": [datetime(2020,1,1),datetime(2020,1,2),datetime(2020,1,3)]
        })

        # If a table is created, the generated sql is returned
        create_statement = fts(
            df,
            table_name,
            connection,
            if_exists="replace",
            clean_cols=True,
            copy=True,
        )

        print("Creating estatement")
        print(create_statement)
        print("Estatement created")

        # Commit upload actions and close connection
        print("Uploading")
        connection.commit()
        print("Uploaded")

        def update_column_names(cursor, table_name, new_column_names):

            cursor = cursor

            # Get the existing column names
            cursor.execute(f"SELECT COLUMN_NAME FROM INFORMATION_SCHEMA.COLUMNS WHERE TABLE_NAME = '{table_name}'")
            columns = [column[0] for column in cursor.fetchall()]

            # Verify that the number of new column names matches the existing number of columns
            if len(new_column_names) != len(columns):

                #st.write("Error: The number of new column names does not match the number of existing columns.")
                print("Error: The number of new column names does not match the number of existing columns.")
                return

            # Generate and execute the ALTER TABLE queries to update column names
            for i in range(len(columns)):

                old_column_name = columns[i]
                print("Old Column Name")
                print(old_column_name)

                new_column_name = new_column_names[i]
                print("New Column Name")
                print(new_column_name)

                # Construct the ALTER TABLE query
                table_column = str(f'[{table_name}].[{old_column_name}]')
                alter_query = f"EXEC sp_rename '{table_column}', '{new_column_name}'"

                # Execute the query
                cursor.execute(alter_query)

            # Commit the changes and close the connection
            cursor.commit()
            cursor.close()
            print("Column names updated")

        columns_to_edit = list(df.columns)
        #update_column_names(
        #    cursor=cursor,
        #    table_name=table_name,
        #    new_column_names=columns_to_edit
        #)


        connection.close()
        print("Connection closed")
    fast_sql_test()
```
